core/server/data/migrations/versions/4.0/01-remove-not-null-constraint-from-plan-name-column.js

```diff
@@ -0,0 +1,64 @@
+const {createNonTransactionalMigration} = require('../../utils');
+const {createTable, deleteTable} = require('../../../schema/commands');
```

Comment:
Not sure if I should be using the commands directly? Like should I create a helper in the utils for this? :thinking:

Reply:
@allouis Just wondering if we should include the whole table deletion/temp table/re-creation dance in the util instead of just these 2 methods. The whole flow for re-creation with correct tableDef is pretty generic and having it out in a util will mean easier to re-use as well as update it if we/knex comes across with a better way in future.

Reply:
Yeah that's what I was thinking, but it might be best to do that if we run into this a second time, not sure.

@daniellockyer what do you think?

Reply:
Yeah I'd say leave it for now and implement it if we need it again in the future. Are we likely to see this again soon?

Reply:
A util for whole table dance would be good, especially as we'll want one for dropping columns too, but I don't think it's a blocker to getting this PR merged.